meganotes-master/app/notes/notes.service.js

```diff
@@ -0,0 +1,71 @@
+(function() {
+  angular.module('meganote.notes')
+    .service('NotesService', NotesService);
+
+  NotesService.$inject = ['$http'];
+  function NotesService($http) {
+    var service = this;
+    service.notes = [];
+
+    service.getNotes = function() {
+      var notesPromise = $http.get('http://localhost:3030/');
```

Comment:
TODO: Make a .constant for the URL in your module, also make a different file for service